gateway-shell/src/test/java/org/apache/knox/gateway/shell/table/KnoxShellTableTest.java

```diff
@@ -362,11 +362,13 @@ public Object answer() {
   @Test
   public void shouldReturnDifferentCallHistoryForDifferentTables() throws Exception {
     final KnoxShellTable table1 = new KnoxShellTable();
-    table1.id(1);
-    KnoxShellTableCallHistory.getInstance().saveCall(1, new KnoxShellTableCall("class1", "method1", true, Collections.singletonMap("param1", String.class)));
+    final long id1 = KnoxShellTable.getUniqueTableId();
+    table1.id(id1);
```

Comment:
I haven't looked at the implementation but shouldn't a table get a unique ID automatically? Do we need to set an id on the table? We should be able to ask the table for the id?

Reply:
I've to double-check but there might be a chicken/egg issue between the builder and the table object...I'll let you know.